vividus-allure-adaptor/src/main/java/org/vividus/report/allure/plugin/VividusLogoPlugin.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2019-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.report.allure.plugin;
+
+import java.util.List;
+
+import javax.inject.Named;
+
+@Named
+public class VividusLogoPlugin extends EmbeddedPlugin
```

Comment:
Any benefits of having a class? Why not put this into the xml config?

Reply:
The only benefit I see (from my point of view) is that it's easier to configure plugins via java code. Potentially it will result in more simple support:)

Reply:
Can you please declare it in the config then? 
Because I believe "the best code is no code at all." :)